explorer/types.go

```diff
@@ -123,9 +123,32 @@ type SiafundOutput struct {
 	types.SiafundElement
 }
 
-// A FileContract is a types.FileContractElement with added fields for
-// resolved/valid state.
+// A ContractSiacoinOutput is a SiacoinOutput with an added field for its ID.
+type ContractSiacoinOutput struct {
+	ID types.SiacoinOutputID `json:"id"`
+	types.SiacoinOutput
+}
+
+// A FileContract is a storage agreement between a renter and a host. It
+// contains a bidirectional payment channel that resolves as either "valid" or
```

Comment:
I'd rephrase this a bit. Instead of `contains` say `serves as` since the contract is the channel and strictly speaking its only safe to use as a unidirectional payment channel since otherwise the host has incentive to not publish the latest revision which should be the one that contains the highest host payout.

Reply:
I changed it here but also want to say I copied the docstring from `core/types`.